### PR TITLE
AS-81: Move project under Speechify's name, setup CI/CD

### PR DESCRIPTION
**Pull request by GedasGa**
## Description

This project was recently moved from the https://github.com/Saghen account. This PR updates `package.json` and other related places, renames the owner of the repo, and points to @speechify's GitHub packages registry.

Also sets up an automated CI/CD pipeline for testing and deployment.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v18
```
Comment (saghen) [inline .nvmrc]: Can we instead use `.node-version` with `18` instead of `v18`

**Review by saghen — CHANGES_REQUESTED** [review]
A few small things:
-  Can we use `.yaml` for the workflows for consistency with our manifests in other projects
- Publish to @saghen/string-tracker on NPM as well. I've added a token for writing packages as `NPM_TOKEN`

File: .nvmrc (hunk 1 above)
Comment (GedasGa) [reply .nvmrc]: Yeah, of course, we could, I just happen to use `nvm` on my machine, and I think that's the only format it supports. What Node version manager do you use?

Comment (GedasGa) [reply .nvmrc]: > @Saghen Can we use .yaml for the workflows for consistency with our manifests in other projects

I see that for instance in the audio server, we also use `.yml` to define GitHub workflows. Anyway, I've changed the extension to `.yaml`

Comment (saghen) [reply .nvmrc]: Yeah need to change those as well, thanks! I use `fnm` https://github.com/Schniz/fnm

Comment (saghen) [reply .nvmrc]: Looks like there's been an issue open for supporting .node-version for 7 years :smile: https://github.com/nvm-sh/nvm/issues/794

**Review by saghen — APPROVED** [review]

**Pull request merged**